content/numerical/PolynomialBase.h

```diff
@@ -0,0 +1,57 @@
+/**
+ * Author: chilli, Andrew He, Adamant
+ * Date: 2019-04-27
+ * Description: A FFT based Polynomial class.
+ */
+#pragma once
+
+#include "../number-theory/ModularArithmetic.h"
+#include "FastFourierTransform.h"
+#include "FastFourierTransformMod.h"
+// #include "NumberTheoreticTransform.h"
```

Comment:
Nit: NTT will be the common case, if that matters to you?

Reply:
This is currently commented out for a stupid reason -  `NumberTheoreticTransform.h` doesn't compile properly on its own (due to redefinitions of const mod). It is good to know that NTT is the common case though.